terminal/command.go

```diff
@@ -165,26 +266,53 @@ func (c *Commands) Merge(allAliases map[string][]string) {
 	}
 }
 
+var noCmdError = errors.New("command not available")
+
 func noCmdAvailable(t *Term, ctx callContext, args string) error {
-	return fmt.Errorf("command not available")
+	return noCmdError
 }
 
 func nullCommand(t *Term, ctx callContext, args string) error {
 	return nil
 }
 
 func (c *Commands) help(t *Term, ctx callContext, args string) error {
+	if args != "" {
+		for _, cmd := range c.cmds {
+			match := false
+			for _, alias := range cmd.aliases {
+				if alias == args {
+					match = true
```

Comment:
Let's just print the help message here and return instead of breaking and printing below.